docs/src/index.md

```diff
@@ -189,6 +189,10 @@ Depth = 2
 
 ## Library Outline
 
+```@docs
+LazySets
```

Comment:
just a comment: many people do not add a docstring to `module LazySets`, in which case this `@docs` is not needed -- by default julia will print the README.md of the package. i like this idea because it has some more info such as links to online documentation that our current `Main module for `LazySets.jl` -- a Julia package for calculus with convex sets.`

Reply:
Okay, then I vote for removing this `@docs`. Otherwise we get a warning.

Reply:
okay. can you do that?

Reply:
done